Exercises/4-curry.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Check 4 digit pin.
+const checkPin = (...code) =>
+  code.join('') === '4967';
```

Comment:
Single-line function

Reply:
See example below to refactor this code: extract magic number to constant and hold it in closure.